CHANGELOG.md

```diff
@@ -8,6 +8,13 @@ and this project adheres to
 
 ## [Unreleased]
 
+## 0.6.1 - 2020-04-21
```

Comment:
Considering this is technically a new feature in the sdk, it would be more appropriate to do a minor version bump to (0.7.0) instead of a patch.

I tend to just add the version to the change log prior to adding the publish commit to master since there are sometimes a couple of changes making their way into the project all at once.

Reply:
Oh and fyi, I'm stealing 0.6.1 to get a hotfix out

Reply:
The minor version bump makes sense, I was a bit hesitant since it's much smaller than the other minor versions. 
Not sure if you're suggesting just adding changes + additions under the [Unreleased] header until a new version is pushed. If so, I agree.

Reply:
>Not sure if you're suggesting just adding changes + additions under the [Unreleased] header until a new version is pushed. If so, I agree.

Yeah, that's what I'm suggesting.